.goreleaser.yaml

```diff
@@ -69,7 +69,10 @@ brews:
     commit_msg_template: "Brew formula update for {{ .ProjectName }} version {{ .Tag }}"
     homepage: "https://zarf.dev/"
     description: "DevSecOps for Air Gap"
-  - name: "zarf@{{ .Tag }}"
+
+  # NOTE: We are using .Version instead of .Tag because homebrew has weird semver parsing rules and won't be able to
```

Comment:
Technically it isn't weird, real semver doesn't support the v

Reply:
Eh who cares https://semver.org/#is-v123-a-semantic-version